### PR TITLE
Add Stable-Code-3B tutorial

### PR DESCRIPTION
**Pull request by Andrei-Aksionov**
Hi there 👋 

Part of #802

Here I added `stable-code-3b` model and created a missing tutorial for StableCode collection.

This model is for code completion, but with a twist (a weird one): presumably it is paid by the number of lines of code, meaning that it doesn't want to stop generating. Or maybe it just loves to code 🤷‍♂️ .
The same happens with the HF transformers, I copied the code-snippet from the [model's card](https://huggingface.co/stabilityai/stable-code-3b), only changed the prompt slightly and here is the output:
```python
Write in Python a softmax function. Be concise.

# In[ ]:


def softmax(x):
    """Compute softmax values for each sets of scores in x."""
    return np.exp(x) / np.sum(np.exp(x), axis=0)


# In[ ]:


softmax(np.array([0, 0, 0]))


# In[ ]:


softmax(np.array([0, 0, 1]))


# In[ ]:


softmax(np.array([0, 1, 0]))


# In[ ]:


softmax(np.array([0, 1, 1]))


# In[
```

Only the `max_new_tokens` limit stopped it.

### CODE REVIEW

File: lit_gpt/config.py [hunk 1]
```diff
@@ -188,6 +188,53 @@ def norm_class(self) -> Type:
     ),
 ]
 
+
+##########################
```
Comment (carmocca) [inline lit_gpt/config.py:192]: Why did you choose to move this?

**Review by carmocca — APPROVED** [review]
Thank you Andrei

**Comment by carmocca** [comment]
About the model not emitting the stop token, in the past I came up with a list of tokens to use as stop tokens by trial and error, you could do the same: https://github.com/Lightning-AI/lit-gpt/blob/main/chat/base.py#L229-L236

File: lit_gpt/config.py (hunk 1 above)
Comment (Andrei-Aksionov) [reply lit_gpt/config.py:192]: So StableCode is placed right after StableLM.

**Comment by Andrei-Aksionov** [comment]
The process doesn't look particularly engaging, but I'll try :)
Thanks.

**Comment by carmocca** [comment]
Up to you. We can just accept that the model is not good at stopping. I'll merge this in the meantime

**Comment by Andrei-Aksionov** [comment]
> Up to you. We can just accept that the model is not good at stopping. I'll merge this in the meantime

I'll try it after I finish updating StableLM models. 
If I succeed - create a PR.

**Comment by Andrei-Aksionov** [comment]
Btw, it was my 50th commit 🎉.

**Pull request merged**